charts/backbeat/templates/producer/deployment.yaml

```diff
@@ -36,8 +36,16 @@ spec:
           # TODO livenessProbe:
           # TODO readinessProbe:
           resources:
-{{ toYaml .Values.resources | indent 12 }}
-    {{- if .Values.nodeSelector }}
+{{ toYaml .Values.producer.resources | indent 12 }}
+    {{- with .Values.consumer.nodeSelector }}
```

Comment:
It should be ```{{- with .Values.producer.nodeSelector }}``` instead of ```{{- with .Values.consumer.nodeSelector }}```.

Reply:
Good catch! Should've `grep`ed :) Will fix.